src/nvapi_drs.cpp

```diff
@@ -47,7 +56,33 @@ extern "C" {
         if (log::tracing())
             log::trace(n, log::fmt::hnd(hSession), log::fmt::ptr(appName), log::fmt::ptr(phProfile), log::fmt::ptr(pApplication));
 
-        return ExecutableNotFound(str::format(n, " (", str::fromnvus(appName), ")"));
+        if (!phProfile || !pApplication)
+            return InvalidArgument(n);
+
+        switch (pApplication->version) {
+            case NVDRS_APPLICATION_VER_V4:
+                std::memset(pApplication->commandLine, 0, sizeof(pApplication->commandLine));
+                [[fallthrough]];
+            case NVDRS_APPLICATION_VER_V3:
+                pApplication->isCommandLine = 0;
+                pApplication->isMetro = 0;
+                [[fallthrough]];
+            case NVDRS_APPLICATION_VER_V2:
+                std::memset(pApplication->fileInFolder, 0, sizeof(pApplication->fileInFolder));
+                [[fallthrough]];
+            case NVDRS_APPLICATION_VER_V1:
+                std::memset(pApplication->launcher, 0, sizeof(pApplication->fileInFolder));
```

Comment:
/
```suggestion
                std::memset(pApplication->launcher, 0, sizeof(pApplication->launcher));
```

Result is the same, but still..